fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/ListTablesDAO.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ 
+package com.ibm.fhir.database.utils.common;
+
+import com.ibm.fhir.database.utils.api.IDatabaseSupplier;
+
+/**
+ * A DAO to list the tables in a given schema
+ */
+public interface ListTablesDAO extends IDatabaseSupplier<SchemaInfoObject>{
```

Comment:
I couldn't find a reference when searching in the web browser. Is this actually used?

Reply:
It was intended to be the base class for the [Derby/Db2/Postgres]ListTablesForSchema implementations. It's not really necessary because we don't use them in a polymorphic context so I'll remove.